$NetBSD: patch-ab,v 1.3 2005/09/27 11:25:36 dsainty Exp $

Reset defaults for the "Trivial LVP programmer", as featured on the home page
of this variant of the programmer software.

--- picprg.c	2002-05-21 03:20:02.000000000 +1200
+++ picprg.c	2005-09-13 01:45:06.000000000 +1200
@@ -325,12 +339,12 @@
     
   /* Defaults: */
   if (do_defaults) {
-     pconfig.port = 0;			/* /dev/lp port		*/
-     pconfig.VPP_pin = 5;		/* VPP on pin 5		*/
+     pconfig.port = 0x378;			/* /dev/lp port		*/
+     pconfig.VPP_pin = 4;		/* VPP on pin 4		*/
      pconfig.VPP_pol = 1;		/* Normal by default	*/
-     pconfig.VDD_pin = 4;		/* Vdd on pin 4		*/
+     pconfig.VDD_pin = 6;		/* Vdd on pin 6		*/
      pconfig.VDD_pol = 1;		/* Normal by default	*/
-     pconfig.PGM_pin = 6;		/* PGM on pin 6		*/
+     pconfig.PGM_pin = 5;		/* PGM on pin 5		*/
      pconfig.PGM_pol = 1;		/* Normal by default	*/
      pconfig.CLK_pin = 3;		/* Clock on pin 3	*/
      pconfig.alt = 0;                   /* Aisha 4/2/00 */
@@ -338,7 +352,7 @@
      pconfig.DATAO_pin = 2;		/* Data to PIC on pin 2	*/
      pconfig.DATAO_pol = 1;		/* Normal by default	*/
      pconfig.DATAI_pin = 10;		/* Data from pic on p10 */
-     pconfig.DATAI_pol = 0;		/* Inverted by default	*/
+     pconfig.DATAI_pol = 1;		/* Normal by default	*/
      pconfig.IDaddr = 0x2000;		/* Default ID locaiton	*/
      pconfig.EEaddr = 0x2100;		/* Default EE location	*/
      pconfig.CFaddr = 0x2007;		/* Config Fuses		*/
